### ISSUE TITLE
Adding example/"cookiecutter" projects

### ISSUE THREAD

**Issue by rasbt** [amueller/futurepast#4]
I think it would be useful to add some example projects to the repo (not the package itself) so that users could browse through some use cases (and optionally clone and execute them). Maybe for the 2 common scenarios `examples/example_pkg` and `examples/example_pkg_with_subpkgs` or so. 

Here, I think it could be more focussed on typical package setups and readability (in contrast to the more comprehensive unit tests).

I'd be happy to help adding these examples (or a PR to better explain what I mean ;). The API looks great so far btw.

**Comment by amueller** [comment]
Yeah I think examples would be great. I haven't really done any docstrings so far, so that comes first ;)

This is like super early stages. I also want to complete the features in #1.

**Comment by rasbt** [comment]
Sure, I understand; honestly impressive that you came up with this within a day or so. If you'd like some help, just ping me some time when you think it's a good time to add some examples! :)

**Comment by amueller** [comment]
Thanks :) Yeah I was like "I should code today" and that was that ^^
